RecipeManager.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace cookBook
+{
+    internal class RecipeManager
+    {
+        private List<Recipe> recipes = new List<Recipe>();
+
+        public int CountEntries(string name)
+        {
+            return recipes.GroupBy(recipe => recipe.Name).Where(group => group.Count() > 1).Sum(group => group.Count());
+        }
+
+        public Recipe FindSingle(string name)
+        {
+            if (recipes.Any())
+            {
+                var matches = recipes.Where(x => x.Name.Equals(name, StringComparison.OrdinalIgnoreCase)).ToList();
+
+                if (matches.Count() == 0)
+                {
+                    return null;
+                }
+                else if (matches.Count() == 1)
+                {
+                    return recipes.Find(x => x.Name.Equals(name, StringComparison.OrdinalIgnoreCase));
+                }
+                else
+                {
+                    int i = 1;
+                    int selection = 0;
+                    Console.WriteLine("Multiple recipes found - select one\n");
```

Comment:
Coś co możnaby rozwiązać (ale nie teraz, na razie to zostaw) to że mamy rozsiane Console.WriteLine po kilku plikach.
Idealnie by było gdyby wszystko z IO było w jednym pliku.
Ale jeśli będzie taka konieczność to przerobimy później